1.1/nanoserver/sdk/msbuild/Dockerfile

```diff
@@ -28,8 +28,8 @@ ENV DOTNET_SKIP_FIRST_TIME_EXPERIENCE true
 RUN mkdir warmup \
     && cd warmup \
     && dotnet new \
-    # .NET Core SDK preview3 targets 1.0, replace with 1.1 to populate cache
-    && powershell -NoProfile -Command "(Get-Content project.json).replace('1.0.1', '1.1.0').replace('netcoreapp1.0', 'netcoreapp1.1') | Set-Content project.json" \
+    # Projects created with .NET Core SDK preview5 target 1.0, replace with 1.1 to populate cache
+    && powershell -NoProfile -Command "(Get-Content warmup.csproj).replace('1.0.1', '1.1.0').replace('netcoreapp1.0', 'netcoreapp1.1') | Set-Content warmup.csproj" \
```

Comment:
This comment applies to the debian Dockerfile as well, and also to the create-run-publish-app scripts.